### ISSUE TITLE
TypeError: Cannot read properties of null (reading 'useReducer')

### ISSUE THREAD

**Issue by EthanBurkett** [awran5/react-simple-typewriter#9]
Just got this error when trying to use the function

Code used:
```ts
const [text, count] = useTypewriter({
    words: ["Hi, my name's Ethan", "Developer", "Creator"],
    loop: true,
    delaySpeed: 2000,
  });
  return (
    <div>
      <p>{text}</p>
    </div>
  );
```

**Comment by Gabe-Alvess** [comment]
Same problem here! 😞

Code:

 ```ts
import React from "react";
import { useTypewriter, Cursor } from "react-simple-typewriter";

type Props = {};

export default function Profile({}: Props) {
  const [text] = useTypewriter({
    words: ["Developer", "Designer", "Creator"],
    loop: true,
    delaySpeed: 2000,
  });

  return (
    <div>
      <h1>
        <span>{text}</span>
        <Cursor cursorColor="red" />
      </h1>
    </div>
  );
}
```

**Comment by EthanBurkett** [comment]
Sometimes the typewriter will work but then randomly it will throw the error and i have to disable it until I feel like trying again and then boom, it magically works again\

**Comment by Dekita** [comment]
Same issue here, worked completely fine until now. 

Edit: I set the version in my package.json to `4.0.2` instead of `^4.0.2` (which installed the latest `4.0.3`). The issue is 'resolved'. Not sure what broke it for the current version.

**Comment by awran5** [comment]
Hi guys, fixed with [v4.0.5](https://github.com/awran5/react-simple-typewriter/compare/v4.0.4...v4.0.5) Please test and let me know.

**Comment by Gabe-Alvess** [comment]
> Hi guys, fixed with [v4.0.5](https://github.com/awran5/react-simple-typewriter/compare/v4.0.4...v4.0.5) Please test and let me know.

Working just fine for me 😁 
Thank You 👍

**Comment by jessicaan** [comment]
> Hi guys, fixed with [v4.0.5](https://github.com/awran5/react-simple-typewriter/compare/v4.0.4...v4.0.5) Please test and let me know.

Working!! Thanks!
